rfcs/text/0005-host-metric-fields.md

```diff
@@ -74,6 +74,160 @@ Stage 1: Provide a high-level description of example sources of data. This does
 Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
 -->
 
+Example source document from AWS EC2 instance:
+```JSON
+{
+  "_index": "metricbeat-8.0.0-2020.10.13-000001",
+  "_type": "_doc",
+  "_id": "7P6XKnUBnE_5JbVxdPGQ",
+  "_version": 1,
+  "_score": null,
+  "_source": {
+    "@timestamp": "2020-10-15T04:39:00.000Z",
+    "host": {
+      "id": "i-04c1a32c2aace6b40",
+      "name": "i-04c1a32c2aace6b40",
+      "cpu": {
+        "usage": 0.0020056497175091896
```

Comment:
Noting that ECS will need to add support for the `scaled_float` type to support the proposed `host.cpu.usage` field.

I've opened #1033 to capture this need.

Reply:
PR to add support: https://github.com/elastic/ecs/pull/1042

Reply:
And now merged 🙌 🎉